lib/periodicChecks.js

```diff
@@ -80,12 +80,14 @@ const ensureAllPullRequestsAreAssigned = async (context) => {
     );
 
     if (hasMergeConflict(pullRequestData)) {
+      // eslint-disable-next-line no-console
```

Comment:
Remove the `eslint-disable-next-line no-console` from this file.

Reply:
ah, missed it. Done